resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -85,7 +85,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
       Map(SPARK_APP_ID_LABEL -> applicationId(), SPARK_ROLE_LABEL -> SPARK_POD_EXECUTOR_ROLE)
     val configMap = KubernetesClientUtils.buildConfigMap(configMapName, confFilesMap, labels)
     KubernetesUtils.addOwnerReference(driverPod.orNull, Seq(configMap))
-    kubernetesClient.configMaps().create(configMap)
+    kubernetesClient.configMaps().inAnyNamespace().resource(configMap).create()
```

Comment:
Can we create in-namespace ConfigMap? Maybe, it's a separate issue?

Reply:
It will be within the namespace as the [namespace changes](https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md#namespace-changes) says

> Only if the client is left at the default namespace or a call has been made to inAnyNamespace will the item namespace be used.

And we have set the namespace on the item:
> https://github.com/apache/spark/blob/ad2f40cdf423d4c5873ca21716dda464446ca127/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtils.scala#L97

Reply:
Double checked:
<img width="1367" alt="image" src="https://user-images.githubusercontent.com/2017933/192407277-d476449e-1112-40fb-916e-81393855988c.png">